### PR TITLE
po: update Ukrainian translation

### PR DESCRIPTION
**Pull request by yurchor**
Many thanks in advance for merging.

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -437,11 +437,15 @@ msgid "Halt the system while an application is inhibiting this"
...
+#| "Authentication is required to hibernate the system while an application "
+#| "is inhibiting this."
 msgid ""
-"Authentication is required to halt the system while an application asked "
-"to inhibit it."
+"Authentication is required to halt the system while an application is "
+"inhibiting this."
 msgstr ""
-"Потрібна автентифікація, щоб зупинити систему, коли програми намагаються "
+"Потрібна автентифікація, щоб перервати роботу систему, коли програма"
```
Comment (keszybz) [inline po/uk.po]: Is this change warrnated? "зупинити систему" is shorter and more direct.

File: po/uk.po [hunk 2]
```diff
@@ -437,11 +437,15 @@ msgid "Halt the system while an application is inhibiting this"
 msgstr "Зупинити систему, коли програми намагаються перешкодити цьому"
 
 #: src/login/org.freedesktop.login1.policy:247
+#| msgid ""
+#| "Authentication is required to hibernate the system while an application "
+#| "is inhibiting this."
```
Comment (keszybz) [inline po/uk.po]: Please remove the comment.

File: po/uk.po [hunk 3]
```diff
@@ -767,26 +770,36 @@ msgid "Authentication is required to reset DNS settings."
 msgstr "Для скидання параметрів DNS до типових слід пройти розпізнавання."
 
 #: src/network/org.freedesktop.network1.policy:143
+msgid "DHCP server sends force renew message"
+msgstr "Сервер DHCP надсилає повідомлення щодо примусового оновлення"
+
+#: src/network/org.freedesktop.network1.policy:144
+#| msgid "Authentication is required to set a wall message"
```
Comment (keszybz) [inline po/uk.po]: Misplaced comment?

File: po/uk.po (hunk 2 above)
Comment (yurchor) [reply po/uk.po]: This is automated gettext fuzzy matching. Removed.

File: po/uk.po (hunk 3 above)
Comment (yurchor) [reply po/uk.po]: This is automated gettext fuzzy matching. Removed.

File: po/uk.po (hunk 1 above)
Comment (yurchor) [reply po/uk.po]: Fixed. Thanks.

**Pull request merged**